### ISSUE TITLE
Portal does not update `className` and `style` when new props arrive

### ISSUE THREAD

**Issue by sompylasar** [tajo/react-portal#72]
It renders `className` and `style` into the element only once, during creation of the element:
https://github.com/tajo/react-portal/blob/f03eca77da12a8419231fc20c5e2fec181e92176/lib/portal.js#L135
```jsx
  renderPortal(props) {
    if (!this.node) {
      this.node = document.createElement('div');
      if (props.className) {
        this.node.className = props.className;
      }
      if (props.style) {
        CSSPropertyOperations.setValueForStyles(this.node, props.style);
      }
      document.body.appendChild(this.node);
    }
    this.portal = ReactDOM.unstable_renderSubtreeIntoContainer(
      this,
      React.cloneElement(props.children, { closePortal: this.closePortal }),
      this.node,
      this.props.onUpdate
    );
  }
```

If the new values arrive through the `props`, they are simply ignored, only the `children` get updated.

**Comment by tajo** [comment]
I guess it would help to move
```jsx
if (props.style) {
  CSSPropertyOperations.setValueForStyles(this.node, props.style);
}
```
outside that condition (else branch).

Do want to create a PR (and ideally add a test)?

**Comment by sompylasar** [comment]
Done: https://github.com/tajo/react-portal/pull/78

**Comment by tajo** [comment]
Fix released as `v2.2.0`.
